### ISSUE TITLE
README about fedora-cert package.

### ISSUE THREAD

**Issue by junaruga** [pypingou/fedora-active-user#17]
Shall we modify the `README.md` too after the recent source modification?

For example, the part

```
This script depends on the fedora-cert package.
```

in `README.md`

could be like this.

```
This script depends on the fedora-cert package to use full functions optionally.
```

**Comment by pypingou** [comment]
We could, though I'd just say that fedora-cert is an optional dependency.
Similarly, we could add support for https://pagure.io/cccolutils and make it also an optional dependency

**Comment by junaruga** [comment]
Yes I am okay for "I'd just say that fedora-cert is an optional dependency".

> Similarly, we could add support for https://pagure.io/cccolutils and make it also an optional dependency

The support for the cccolutils will be implemented in the future right?
I do not see it in the current code.

And I see other optional dependencies `from fedora.client.bodhi import BodhiClient` and `from fedora.client import AccountSystem`.

```
$ grep -n import fedora_active_user.py 
24:import datetime
25:import getpass
26:import json
27:import logging
28:import re
29:import sys
30:import time
35:    import urllib.parse
36:    import urllib.request
38:    import urllib
72:    from fedora.client.bodhi import BodhiClient
106:    from bugzilla import Bugzilla
169:    import koji
296:    from fedora.client import AccountSystem
301:        import fedora_cert
511:    import argparse
543:    import sys
```

Possibly those could be documented. and the test with the dependencies could be included for the Travis CI Fedora 31/30 cases.
